components/google_drive/actions/create-file/create-file.js

```diff
@@ -0,0 +1,244 @@
+const googleDrive = require("../../google_drive.app");
+const fs = require("fs");
+const got = require("got");
+
+module.exports = {
+  key: "google_drive-create-file",
+  name: "Create a New File",
+  description: "Create a new file from a URL or /tmp/filepath.",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    googleDrive,
+    drive: {
+      propDefinition: [
+        googleDrive,
+        "watchedDrive",
+      ],
+    },
+    uploadType: {
+      type: "string",
+      label: "Upload Type",
+      description: `The type of upload request to the /upload URI. If you are uploading data (using an /upload URI), this field is required. If you are creating a metadata-only file, this field is not required. 
+        media - Simple upload. Upload the media only, without any metadata.
+        multipart - Multipart upload. Upload both the media and its metadata, in a single request.
+        resumable - Resumable upload. Upload the file in a resumable fashion, using a series of at least two requests where the first request includes the metadata.
+      `,
+      options: [
+        "media",
+        "multipart",
+        "resumable",
+      ],
+    },
+    fileUrl: {
+      type: "string",
+      label: "File URL",
+      description:
+        "The URL of the file you want to upload to Google Drive. Must specify either File URL or File Path.",
+      optional: true,
+    },
+    filePath: {
+      type: "string",
+      label: "File Path",
+      description:
+        "The path to the file, e.g. /tmp/myFile.csv . Must specify either File URL or File Path.",
+      optional: true,
+    },
+    ignoreDefaultVisibility: {
+      type: "boolean",
+      label: "Ignore Default Visibility",
+      description: `Whether to ignore the domain's default visibility settings for the created file. Domain administrators can choose to make all uploaded files visible to the domain by default; this parameter bypasses that behavior for the request. Permissions are still inherited from parent folders. 
+        (Default: false)
+      `,
+      optional: true,
```

Comment:
**-Minor-**
Explicitly setting the default value via the prop definition improves the UX (in my opinion), since it doesn't require the user to read the whole description and it also auto-populates the field in the UI, so the user will be inclined to accept this fact and proceed to the next prop without too much friction.

```suggestion
      `,
      default: false,
```

Reply:
Updated